activerecord/lib/active_record/railtie.rb

```diff
@@ -368,7 +370,7 @@ class Railtie < Rails::Railtie # :nodoc:
           ActiveRecord.query_transformers << ActiveRecord::QueryLogs
           ActiveRecord::QueryLogs.taggings.merge!(
             application:  Rails.application.class.name.split("::").first,
-            pid:          -> { Process.pid },
+            pid:          -> { Process.pid.to_s },
```

Comment:
I'm casting this to a string, since that's the desired type for our query log values. This removes any question about whether it gets coerced into a string later on.